src/components/avatar/index.js

```diff
@@ -28,6 +29,14 @@ export default class Avatar extends BaseComponent {
   static displayName = 'Avatar';
   static modes = STATUS_MODES;
   static propTypes = {
+    /**
+     * Adds fade in animation when Avatar image loads
+     */
+    animate: PropTypes.bool,
+    /**
+     * Duration of the fade in animation when the Avatar image loads
+     */
+    animationDuration: PropTypes.number,
```

Comment:
I would expect this prop to be passed using `imageProps` since it's part of `AnimatedImage`'s props